.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,17 @@
+name: Build and Test
+on: [push]
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "This job was triggered by a ${{ github.event_name }} event."
+      - run: echo "Running on OS ${{ runner.os }}"
+      - run: echo "Checking out branch ${{ github.ref }}."
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - name: "Run `bazel build`"
+        run: |
+          bazel build //...
```

Comment:
Is bazel already installed? I don't *think* it is, and you might need a bazel installation action

https://github.com/bazelbuild/setup-bazelisk maybe this?

Reply:
It is installed! https://github.com/actions/runner-images/pull/490

Reply:
thanks!